cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1376,7 +1377,7 @@ private void handleSubscribe(boolean newSubscription, SubscribedCallbackListener
         }
       }
 
-      if (!this.areNotificationsEnabled() && !this.ignoreDisabledNotificationPermission) {
```

Comment:
This is not required. Only change logic in `requestNotificationPermission` logic, not here.

Reply:
If notification is not enabled and ignoreDisabledNotificationPermission is also false then also subscribe will not perform. It will return from there.

Reply:
@gbhrdt I have tested after removing shouldAutoRequestNotificationPermission from this if. 

requestNotificationPermission dialog is not opening but also subscribe is not working.

<img width="1536" alt="Screenshot 2023-11-27 at 2 00 25 PM" src="https://github.com/cleverpush/cleverpush-android-sdk/assets/129381063/440ec9c8-abcb-4b8f-a734-6f2afd61fb52">